CHANGELOG.md

```diff
@@ -71,6 +71,26 @@ Due to a specification change `write_timestamp` is no longer supported on WebGPU
 
 By @wumpf in [#5188](https://github.com/gfx-rs/wgpu/pull/5188)
 
+#### Querying shader compilation errors
+
+Wgpu now supports querying [shader compilation info](https://www.w3.org/TR/webgpu/#dom-gpushadermodule-getcompilationinfo).
+
+This allows you to get more structured information about compilation errors, warnings and info:
+```rust
+...
+let lighting_shader = ctx.device.create_shader_module(include_wgsl!("lighting.wgsl"));
+let compilation_info = lighting_shader.get_compilation_info().await;
+for message in compilation_info.messages.iter() {
+    let line_number_to_highlight = message
+        .location
+        .map(|location| location.line_number)
+        .unwrap_or(1);
+    println!("Compile error at line {line_number_to_highlight}");
```

Comment:
not all messages are errors, so this is a bit misleading

Reply:
Good point, thank you for catching this